service/policy/attributes/attributes.go

```diff
@@ -96,9 +96,17 @@ func (s *AttributesService) GetAttribute(ctx context.Context,
 ) (*connect.Response[attributes.GetAttributeResponse], error) {
 	rsp := &attributes.GetAttributeResponse{}
 
-	item, err := s.dbClient.GetAttribute(ctx, req.Msg.GetId())
+	var identifier any
+
+	if req.Msg.GetId() != "" { //nolint:staticcheck // Id can still be used until removed
+		identifier = req.Msg.GetId() //nolint:staticcheck // Id can still be used until removed
```

Comment:
I don't understand why we need this if it's only running in the platform. Is this related to ConnectRPC and legacy gRPC requests?

Reply:
No this is because I moved the fields to a oneof in the proto definition. So the id field would be removed in a later release. 

The intention of the oneof was so a user could only set either x_id or fqn/name. But after syncing with @jakedoublev it might not be worth the extra effort to move to the oneof.